File: unn/unn.pl
#!/usr/bin/env perl
use strict;
use warnings;
my $debug = 0;
my @files = ();
foreach (@ARGV)
{
	if ($_ eq '-debug')
	{
		$debug = 1;
	}
	else
	{
		@files = (@files,$_);
	}
}

foreach (@files)
{
	if ( -e $_ )
	{
		print STDERR "fatal error: $_ exists\n"
	}
	else
	{
		if ($debug)
		{
			print "starting execution of $_ ...\n";
            print  "we are done\n";
		}
	}
}

File: unn/unn.php
#!/usr/bin/env php
<?php
	$files = array();
	$debug = false;
	//eliminate calling command from list
	array_shift($argv);
	// loop through the arguments 
	foreach ($argv as $arg)
	{
		if ($arg == '-debug')
		{
			$debug = true;
		}
		else
		{
			$files[] = $arg;
		}
	}
	// loop through the files
	foreach ($files as $file)
	{
		if (file_exists($file))
		{
			fwrite(STDERR,"fatal error: $file exists\n");
		}
		else
		{
			if ($debug)
			{
				echo "starting execution of $file ...\n";
				echo "we are done\n";
			}
		}
	}
?>